benshidanny11/Papel
# Check account details feature

#### What does this PR do?
- It adds view account details feature to the system
#### Description of Task to be completed?
- To create an API endpoint for account details
#### How should this be manually tested?
- Through this [link](https://github.com/benshidanny11/Papel/pull/new/ft-check-account-details-171166348)
#### What are the relevant pivotal tracker stories?
- #171166348
